lib/modules/datasource/github-releases/index.spec.ts

```diff
@@ -33,10 +20,17 @@ describe('modules/datasource/github-releases/index', () => {
 
   describe('getReleases', () => {
     it('returns releases', async () => {
-      httpMock
-        .scope(githubApiHost)
-        .get('/repos/some/dep/releases?per_page=100')
-        .reply(200, responseBody);
+      jest.spyOn(githubGraphql, 'queryReleases').mockResolvedValueOnce([
+        { version: 'a', releaseTimestamp: '2020-03-09T13:00:00Z' },
+        { version: 'v', releaseTimestamp: '2020-03-09T12:00:00Z' },
+        { version: '1.0.0', releaseTimestamp: '2020-03-09T11:00:00Z' },
+        { version: 'v1.1.0', releaseTimestamp: '2020-03-09T10:00:00Z' },
+        {
+          version: '2.0.0',
+          releaseTimestamp: '2020-04-09T10:00:00Z',
+          isStable: false,
+        },
+      ] as never);
```

Comment:
can we use partial instead, otherwise typescript can't detect required type changes